src/src/AppContext.js

```diff
@@ -45,6 +54,12 @@ function AppProvider({ children }) {
 
   const { addCapability } = useCapabilities();
 
+  async function addNewCapability(name, description) {
+    addCapability(name, description);
+    await sleep(3000);
```

Comment:
Seems hack-ish with the sleep here. White it'll do for a hotfix, maybe it would be worth triggering loadMyProfile on a .then after addCapability returns a response. Assuming addCapability only returns when a dbinsert has occurred and is not eventually consistent 😅.

Reply:
I merely added back the old code that was removed a while ago.

I agree with the sleep, but I couldn't make it work last night with other triggers, so I though 'meh' and decided that this would be good enough for now.